mesonbuild/scripts/depfixer.py

```diff
@@ -406,10 +406,10 @@ def fix_darwin(fname: str, rpath_dirs_to_remove: T.Set[bytes], new_rpath: str, f
         return
     new_rpaths: OrderedSet[str] = OrderedSet()
     if new_rpath:
-        new_rpaths.update(new_rpath)
+        new_rpaths.add(new_rpath)
```

Comment:
I suspect that this is wrong and have a different fix but have been waiting for my macOS expert to arrive.